src/sql/platform/connection/common/connectionStatusManager.ts

```diff
@@ -7,15 +7,23 @@ import { ConnectionManagementInfo } from 'sql/platform/connection/common/connect
 import { ICapabilitiesService } from 'sql/platform/capabilities/common/capabilitiesService';
 import { ConnectionProfile } from 'sql/platform/connection/common/connectionProfile';
 import { IConnectionProfile } from 'sql/platform/connection/common/interfaces';
+import { StopWatch } from 'vs/base/common/stopwatch';
```

Comment:
curious: why move this?

Reply:
I don't recall doing this on purpose, could have been VS Code moving stuff around with the auto-import